doc/16-upgrading-icinga-2.md

```diff
@@ -8,6 +8,33 @@ Specific version upgrades are described below. Please note that version
 updates are incremental. An upgrade from v2.6 to v2.8 requires to
 follow the instructions for v2.7 too.
 
+## Upgrading to v2.14 <a id="upgrading-to-2-14"></a>
+
+### Dependencies and Redundancy Groups <a id="upgrading-to-2-14-dependencies"></a>
+
+Before Icinga v2.12 all dependecies were cumulative.
+I.e. the parent was considered reachable only if no dependency was violated.
+In v2.12 and v2.13, all dependecies were redundant.
+I.e. the parent was considered unreachable only if no dependency was fulfilled.
+v2.14 restores the pre-v2.12 behavior and allows to override it.
```

Comment:
Wait a second, doesn't this mix up parent and child? The child has a dependency on the parent, so the child is what can be (un)reachable.